akka-stream/src/main/scala/akka/stream/io/Framing.scala

```diff
@@ -141,21 +124,17 @@ object Framing {
     private val firstSeparatorByte = separatorBytes.head
     private var buffer = ByteString.empty
     private var nextPossibleMatch = 0
-    private var finishing = false
```

Comment:
This wasn't used. There was no way of telling if it should be used, so I deleted it.